src/aind_slims_api/core.py

```diff
@@ -199,6 +204,44 @@ def fetch(
 
         return records
 
+    def fetch_models(
+        self,
+        model: Type[SlimsBaseModelTypeVar],
+        *args,
+        sort: Optional[str | list[str]] = None,
+        start: Optional[int] = None,
+        end: Optional[int] = None,
+        **kwargs,
+    ) -> tuple[list[SlimsBaseModelTypeVar], list[dict[str, Any]]]:
+        """Fetch records from SLIMS and return them as SlimsBaseModel objects
+
+        Returns
+        -------
+        tuple:
+            list:
+                Validated SlimsBaseModel objects
+            list:
+                Dictionaries representations of objects that failed validation
+        """
+        response = self.fetch(
+            model._slims_table.default,  # TODO: consider changing fetch method
```

Comment:
model._slims_table should be a str, right?
`model._slims_table,`

Reply:
I think this is a consequence of the way SlimsBaseModel has it's table definition defined.